testing/tests/DevExpress.ui.widgets/pager.tests.js

```diff
@@ -1384,70 +1506,75 @@ QUnit.module('Pager', {
             pageCount: 10,
             pageSizes: [5, 10, 20],
             showNavigationButtons: true,
+            pageIndex: 10,
             pagesCountText: 'of',
             lightModeEnabled: true
         });
 
         let isPageChanged;
         const instance = $pager.dxPager('instance');
 
-        instance.option('pageIndex', 10);
         instance.pageIndexChanged = function() {
             isPageChanged = true;
         };
-
+        instance._useDefaultOptionUpdate = true;
         const $button = $('.dx-next-button');
         $button.click();
 
         assert.ok($button.hasClass('dx-button-disable'));
         assert.ok(!isPageChanged);
     });
 
-    QUnit.test('Pager is not re-rendered in the Light mode when width is not changed', function(assert) {
-        const pager = $('#container')
-            .width(PAGER_LIGHT_MODE_WIDTH)
-            .dxPager({
-                maxPagesCount: 8,
-                pageCount: 10,
-                pageSizes: [5, 10, 20],
-                showInfo: true,
-                totalCount: 86,
-                infoText: 'Page {0} of {1} ({2} items)',
-                pagesCountText: 'of'
-            }).dxPager('instance');
-
-        const spy = sinon.spy(pager, '_renderContentImpl');
+    if(!isRenovatedComponent) {
+        QUnit.test('ToRemove: Rewrite private API _renderContentImpl Pager is not re-rendered in the Light mode when width is not changed', function(assert) {
```

Comment:
Strange to see task in test title. It will be better to change it to TODO comment or rewrite this test now to check rerender by children element references.